script/build-artifact.sh

```diff
@@ -56,7 +56,7 @@ cp "${BINDIR}"/oidc-discovery-provider "${EXTRAS_STAGING}"/bin
 
 # Create the tarballs and checksums
 mkdir -p "${OUTDIR}"
-tar -cvzf "${TARBALL}" --directory "${TMPDIR}/spire" "${TAROPTS[@]}" .
+(cd "${TMPDIR}/spire"; tar -cvzf "${TARBALL}" "${TAROPTS[@]}" ./*)
```

Comment:
nitpicky, but can we do the other option suggested by shellcheck? The current method still leads to each entry being appended with `./`, which isn't a huge deal, but I'd rather just steer clear:
```suggestion
(cd "${TMPDIR}/spire"; tar -cvzf "${TARBALL}" "${TAROPTS[@]}" -- *)
```